server/midddleware/auth.go

```diff
@@ -100,7 +101,16 @@ func AuthFunction(ctx context.Context, jwtValidator *validator.Validator, config
 			}
 		}
 	}()
-
+	// disable health check authn/z
+	var fullMethodName string
```

Comment:
move this in a helper method so that you can simply call it here whether to run auth check or not.

Reply:
refactored. thanks